invenio_records_editor/views.py

```diff
@@ -25,14 +26,43 @@ def create_editor_blueprint(app):
         static_folder="static",
     )
 
-    @blueprint.route("/")
+    @blueprint.route("/<string:rec_type>/<int:recid>")
     @need_editor_permissions("editor-view")
-    def index():
+    def edit_record(rec_type, recid):
         """Render a basic view, with dummy permission editor-view."""
+        ui_config = {}
+        json_editor_config = current_app.config["RECORDS_EDITOR_UI_CONFIG"]
+        endpoint_config = json_editor_config.get(rec_type)
+
+        ui_config.update(endpoint_config)
+        ui_config["record_config"]["apiUrl"] = "{0}{1}".format(
+            endpoint_config["record_config"]["apiUrl"], recid
+        )
+
+        return render_template(
+            app.config["RECORDS_EDITOR_TEMPLATE"],
+            editor_url=app.config["RECORDS_EDITOR_URL_PREFIX"],
+            module_name="Invenio-Records-Editor",
```

Comment:
Is `module_name` the title of the page? In this case, I would rename the param...